GBZG/View/Component/GbzgButton.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  GbzgButton.swift
```

Comment:
GZBG 대문자로 브랜드 절대 지켜~

Reply:
G사의 경우 아래와 같이 abbreviation에 대해 하나의 일반명사로 취급하길 권장하긴 하는데 우리는 어떻게 할까요?

> prefer to capitalize abbreviations as single words, e.g., StartRpc() rather than StartRPC().

Reply:
저 같은 경우는 GBZG라고 대문자를 썼는데 얘기를 해보고 통일해야될 부분 같군요 :)

Reply:
이건 뭐 답이 없는 문제니 편한대로 가보자고~

Reply:
RPC 는 리얼펀치컴퍼니 약자인데요? G사는 어디죠?

Reply:
Xoogle요 ㅋㅋㅋㅋㅋ

Reply:
G사 안되겠네요~

Reply:
일단 우리는 abbreviation에 대해서 다 대문자로 갑시다! GBZG